extensions/azurecore/src/azureResource/constants.ts

```diff
@@ -20,7 +20,8 @@ export enum AzureResourceItemType {
 	message = 'azure.resource.itemType.message',
 	azureMonitor = 'azure.resource.itemType.azureMonitor',
 	azureMonitorContainer = 'azure.resource.itemType.azureMonitorContainer',
-	cosmosDBMongoAccount = 'azure.resource.itemType.cosmosDBMongoAccount'
+	cosmosDBMongoAccount = 'azure.resource.itemType.cosmosDBMongoAccount',
+	cosmosDBMongoCluster = 'azure.resource.itemType.cosmosDBMongoCluster'
```

Comment:
You may remove this if it's not used since you've combined the query.

Reply:
Richt it was used in some code that has been removed, however we should probably keep it since we might want to use it for TreeItem.contextValue at some point, just didn't want to conditionally assign it yet because it might eventually break something else.